index.html

```diff
@@ -41,10 +41,20 @@ <h2>Why you should use it</h2>
 			</div>
 		</div>
 		<div class="row">
+			<div class="col-sm-4 feature">
+				<h3>Modern Features</h3>
+				<p>Push notifications, link previews, new message markers, and more bring IRC to the 21st century.</p>
+			</div>
 			<div class="col-sm-4 feature">
 				<h3>Always Connected</h3>
-				<p>The Lounge never sleeps. Just sign in and pick up where you left off.</p>
+				<p>Remains connected to IRC servers while you are offline. Forget about bouncer software.</p>
```

Comment:
I feel like it's a bit more colloquial to say "Forget about bouncers.". Thoughts?